drivers/storage/portworx/components_test.go

```diff
@@ -11647,8 +11647,6 @@ func TestPodDisruptionBudgetEnabled(t *testing.T) {
 		// Node that does not return a scheduler node name, does not get counted towards quorum calculation as we cannot
 		// determine if it is part of the current k8s cluster or not.
 		{NonQuorumMember: false, NodeLabels: map[string]string{pxutil.NodeLabelPortworxVersion: "3.1.0"}},
-		// Node that is not part of the cluster, does not get counted towards quorum calculation.
```

Comment:
Didn't we have to mock the ClusterDomainServer in this test? It should always be calling the enumerate API, right?